plugins/argo-apps-viz.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: argo-apps-viz
+spec:
+  version: v0.1.0
+  platforms:
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/syndlex/argo-apps-viz/releases/download/v0.1.0/argo-apps-viz_linux_amd64.tar.gz
+    sha256: f321f594a8bdacdcff5bfef76c8c7f99cd9bc46cd035d962badee70d44489f9e
+    files:
+    - from: "./argo-apps-viz"
+      to: "."
+    - from: LICENSE
+      to: "."
+    bin: "argo-apps-viz"
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/syndlex/argo-apps-viz/releases/download/v0.1.0/argo-apps-viz_darwin_amd64.tar.gz
+    sha256: c5422a5c1344245e6cb6dc7656143aea1c7b58c7d3b6ce47e348b6b1554ebd97
+    files:
+    - from: "./argo-apps-viz"
+      to: "."
+    - from: LICENSE
+      to: "."
+    bin: "argo-apps-viz"
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    uri: https://github.com/syndlex/argo-apps-viz/releases/download/v0.1.0/argo-apps-viz_windows_amd64.zip
+    sha256: ecaedbb3526ea6cff292871b08031a76fba7f7ca863b787de6fd2308ee91fa03
+    files:
+    - from: "/argo-apps-viz.exe"
+      to: "."
+    - from: LICENSE
+      to: "."
+    bin: "argo-apps-viz.exe"
+  homepage: https://github.com/syndlex/argo-apps-viz
+  caveats: |
+    Usage:
+      $ kubectl argo-apps-viz
+    For additional options:
+      $ kubectl argo-apps-viz --help
+      or https://github.com/syndlex/argo-apps-viz/blob/v0.1.0/doc/USAGE.md
+  shortDescription: A kubectl plugin that visualizes ArgoCD concepts like the 'App of Apps' and dependency trees for documentation and teaching.
```

Comment:
Please cap at 55 chars. Things like
"A kubectl plugin that" are very obvious in this context already.